frontend/public/icon/google.svg

```diff
@@ -0,0 +1,12 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" xmlns:xlink="http://www.w3.org/1999/xlink" style="display: block;">
```

Comment:
Please move this file to
frontend/assets/icons/...

Reply:
done